link/link.go

```diff
@@ -97,6 +102,26 @@ type RawLinkInfo struct {
 	Type    Type
 	ID      ID
 	Program ebpf.ProgramID
+	Extra   interface{}
+}
+
+func (r RawLinkInfo) ExtraRawTP() *sys.RawTPExtraLinkInfo {
```

Comment:
This should be `(*RawTracepointLinkInfo, bool)` so to that it's possible to type assert without panicking.

Types from `sys` are awkward for users, since they can't import `sys`. So we might have to have aliases for the extra link info.

Reply:
Fixed.
I think we don't need to add aliases but let me know if we need to add.